php/base/Exchange.php

```diff
@@ -1346,6 +1347,15 @@ function ($curl, $header) use (&$response_headers, &$http_status_text) {
         $curl_error = curl_error($this->curl);
         $http_status_code = curl_getinfo($this->curl, CURLINFO_HTTP_CODE);
 
+        // Save cookies in mem here but do not close the handle
+        $cookie_file = static::safe_value($this->curl_options, CURLOPT_COOKIEJAR);
+        $this->last_response_cookies = curl_getinfo($this->curl, CURLINFO_COOKIELIST);
```

Comment:
here we can save it in-mem (the fopen code is only run if the user specifies `CURLOPT_COOKIEJAR`)